tf_encrypted/keras/__init__.py

```diff
@@ -5,11 +5,13 @@
 from tf_encrypted.keras import models
 from tf_encrypted.keras import layers
 from tf_encrypted.keras.models import Sequential
+from tf_encrypted.keras import backend
 
 
 __all__ = [
     'engine',
     'models',
     'layers',
     'Sequential',
+    'backend'
```

Comment:
list not sorted; make sure each line has a `,` at the end (this leaves all existing lines unchanged when more are added)